testing/test_conftest.py

```diff
@@ -10,9 +10,7 @@
 
 @pytest.fixture(scope="module", params=["global", "inpackage"])
 def basedir(request, tmpdir_factory):
-    from _pytest.tmpdir import tmpdir
-
-    tmpdir = tmpdir(request, tmpdir_factory)
+    tmpdir = tmpdir_factory.mktemp("basedir", numbered=True)
```

Comment:
could this also just take the `tmpdir` fixture as a positional?

Reply:
Not because it is a `module`-scoped fixture, and `tmpdir` is `function`-scoped. 😉

Reply:
oh right of course!